js/Course.js

```diff
@@ -15,12 +15,12 @@ function Course(quarter, name, nameID, gt, units,
                 days, hour1, min1, hour2, min2, bldg, room) {
 
     //General information
-    this.quarter = quarter; //written as a string in the format of "season####"
+    this.quarter = (quarter === "" ? "None provided" : quarter); //written as a string in the format of "season####"
```

Comment:
CourseParser.js assigns "" to quarter if the regex for finding the quarter fails. We can be pretty sure that they will both be strings.